.eslintrc

```diff
@@ -34,7 +34,8 @@
         "jsx-a11y/interactive-supports-focus": "off",
         "jsx-a11y/click-events-have-key-events": "off",
         "react-hooks/rules-of-hooks": "error",
-        "react-hooks/exhaustive-deps": "warn"
+        "react-hooks/exhaustive-deps": "warn",
+        "no-console": "error"
```

Comment:
Kunne denne vært warning? Da vil man få feil når man forsøker å committe, men likevel få lov til å logge ut for debugging

Reply:
[66e309b](https://github.com/navikt/familie-klage-frontend/pull/333/commits/66e309b38739773db5f4ec335f2c4ff1d5fe1e3e)